src/tutorials/code/node/interactive/progress/2.js

```diff
@@ -0,0 +1,3 @@
+let json = {"tactile": [new Packets.ProgressUpdate.TactileUpdate({"id": 0, "cooldown": 30000})]};
+let update = Packets.ProgressUpdate(json);
+robot.send(Packets.ProgressUpdate(update));
```

Comment:
No, this is wrong. 

```
const update = new Packets.ProgressUpdate(json);
robot.send(update);
```
Would be correct, @ACPixel the new is important. We had this discussion in gitter yesterday :'(